plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/CreateMavenBasedAppEngineStandardProject.java

```diff
@@ -59,15 +62,12 @@ protected void execute(IProgressMonitor monitor)
     properties.put("application-id", appId);
 
     ProjectImportConfiguration importConfiguration = new ProjectImportConfiguration();
-    List<IProject> archetypeProjects = projectConfigurationManager
-        .createArchetypeProjects(location, getArchetypeDescriptor(), groupId, artifactId,
-            "0.0.1-SNAPSHOT",
-            packageName, properties, importConfiguration, progress.newChild(60));
-
-    if (!archetypeProjects.isEmpty()) {
-      Job job = new MappingDiscoveryJob(archetypeProjects);
-      job.schedule();
-    }
+    List<IProject> archetypeProjects = projectConfigurationManager.createArchetypeProjects(location,
+        getArchetypeDescriptor(), groupId, artifactId, version, packageName, properties,
+        importConfiguration, progress.newChild(60));
+
+    Job job = new MappingDiscoveryJob(archetypeProjects);
```

Comment:
Added a note.  I've looked for alternatives and there's no API-sanctioned way to trigger the lifecycle-mappings discovery.  But I wonder if we actually need to do this since we know the content of the archetype and have set up lifecycle mappings for the `versions-maven-plugin`.